libraries/commerce/favorites/streams/index.js

```diff
@@ -47,3 +49,20 @@ export const favoritesWillRemoveItem$ = main$
   .filter(({ action }) => action.type === REQUEST_REMOVE_FAVORITES && !action.silent);
 
 export const favoritesSyncIdle$ = main$.filter(({ action }) => action.type === IDLE_SYNC_FAVORITES);
+
+export const addRemoveFavorites$ = main$.filter(({ action }) => (
```

Comment:
Since that stream is the combination of REQUEST_ADD_FAVORITES and REQUEST_REMOVE_FAVORITES (that are not silent) actions it is not a sync. I changed the name of the stream to didAddOrRemoveFavorites$